src/regularTasks.ts

```diff
@@ -187,8 +206,8 @@ const GLOBAL_TASKS: CandyTask[] = [
     do: () => adv1(drunkSafeWander("wanderer"), -1, ""),
     prepare: () =>
       shouldRedigitize() && SourceTerminal.educate([$skill`Digitize`, $skill`Extract`]),
-    post: () =>
-      get("_sourceTerminalDigitizeMonsterCount") || (CandyState.digitizeInitialized = false),
+    post: () => get("_sourceTerminalDigitizeMonsterCount") || (digitizeInitialized = false),
+    sobriety: "sober",
```

Comment:
I'm not sure if this is accurate; we can only _cast_ digitize when sober, but we can absolutely fight digitized wanderers when sober

Reply:
I'll punt it for now.